google/cloud/edgenetwork/README.md

```diff
@@ -22,7 +22,7 @@ this library.
 #include <iostream>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 3) {
+  if (argc != 4) {
     std::cerr << "Usage: " << argv[0] << " project-id location-id\n";
```

Comment:
There is now a mismatch between the requirement and the error message.

Reply:
Thanks! Fixed.